### ISSUE TITLE
Change the control type announced by the VoiceOver screen reader when it encounters a Maui Picker control

### ISSUE THREAD

**Issue by gbarkerz** [dotnet/maui#7001]
### Description

Currently when the VoiceOver screen reader encounters a Maui Picker control, its announcement includes "Text field, double-tap to edit". This gives the impression to customers that they've encountered a text field, and if they double-tap, they can edit the contents of the field. This VoiceOver announcement does not seem to appropriately convey the semantics of the Picker control. 

My feature request here is that VoiceOver's announcement includes a control type which is a closer match to the semantics of the control. For example, "Popup button", given that when invoked, the Picker shows a popup window containing the items in the Picker.   

### Public API Changes

No public API change.

### Intended Use-Case

A few months ago I built a Xaramin app with a Picker control. I asked a user who is blind to try out my app and provide me with feedback. This user used the VoiceOver screen reader. His feedback was as follows:

_The control I double tap on to make the adjustable control appear is actually being reported as an editable control, but should be reported as a type of control called a "popup button."  Windows does not have this control type, but Mac and IOS do.  On MacOS, Popup buttons are used in many of the same places Windows would use combo boxes for.  On IOS, popup buttons aren’t as widely used, in fact I think ideally you could just stick the adjustable control right in the middle of the sheet and you’d be fine.  That may be something Xamarin doesn’t allow for, but it should let the button be a popup button._

Based on this, it seems likely that to change the announced control type would be helpful to customers.

**Comment by RohitJadhav-TCS** [comment]
GitHubTags:
#.NET MAUI; #AllySev3; #A11yMAS; #A11yTCS; #Benchmark; #DesktopApp; #Win11; #WCAG4.1.2; #GH_.NETMAUI_Win32_Apr2022; #AILimited​; #Win32; #VoiceOver;

**Comment by 12-shweta** [comment]
GitHubTags:
#.NET MAUI; #A11ySev3; #A11yMAS; #A11yTCS; #Benchmark; #DesktopApp; #Win11; #WCAG4.1.2; #GH_.NETMAUI_Win32_Apr2022; #AILimited​; #Win32; #VoiceOver;

**Comment by gbarkerz** [comment]
Thanks @PureWeen, that's good to know.
